api/src/Microsoft.Azure.IIoT.OpcUa.Api/src/Core/Models/SimpleAttributeOperandApiModel.cs

```diff
@@ -39,5 +39,12 @@ public class SimpleAttributeOperandApiModel {
         [DataMember(Name = "indexRange", Order = 3,
             EmitDefaultValue = false)]
         public string IndexRange { get; set; }
+
+        /// <summary>
+        /// Optional display name
+        /// </summary>
+        [DataMember(Name = "displayName", Order = 4,
+            EmitDefaultValue = false)]
+        public string DisplayName { get; set; }
```

Comment:
>public string DisplayName { get; set; } [](http://example.com/codeflow?start=8&length=39)

Do we really need 'DisplayName'? It's never used and SimpleAttributeOperand in OpcUA spec does not contain a DisplayName field